src/components/List/ApplicationsList.tsx

```diff
@@ -1,12 +1,15 @@
 import React, { Fragment } from 'react'
 import { Table, Message, Popup } from 'semantic-ui-react'
 import { useLanguageProvider } from '../../contexts/Localisation'
-import { useDeleteApplicationMutation } from '../../utils/generated/graphql'
+import { ApplicationStatus, useDeleteApplicationMutation } from '../../utils/generated/graphql'
 import { ApplicationListRow, ColumnDetails, SortQuery } from '../../utils/types'
 import Loading from '../Loading'
 import { TableCellMobileLabelWrapper } from '../../utils/tables/TableCellMobileLabelWrapper'
 import { useViewport } from '../../contexts/ViewportState'
 
+// Which column in the column array is the "Action" cell
+const ACTION_CELL_INDEX = 6
```

Comment:
This works because we know the structure of the Applications table. It would be a lot more complicated for a generic table (like the Data Views).